scripts/run.yaml

```diff
@@ -14,3 +14,13 @@ runs:
     command: /usr/bin/sqlite3 
     memory: 64
     networking: False
+  - name: node
+    rootfs: ../dynamic-apps/node
+    command: /bin/node
+    memory: 512
+    networking: True
+  - name: gnupg
+    rootfs: ../dynamic-apps/gnupg
+    command: /usr/bin/gpg
```

Comment:
Add argument to `gpg` command for actual functionality.

Reply:
I've added an issue about NodeJS hanging at https://github.com/unikraft/dynamic-apps/issues/79

For "Add argument to gpg command for actual functionality.", may I ask what do you mean by "actual functionality"? cuz gpg don't provide a shell and require user-defined arguments? Do you mean a --version argument?